scripts/gfg.js

```diff
@@ -13,24 +13,24 @@ const UPDATE_MSG = 'Updated solution - LeetHub';
 let START_MONITOR = true;
 const toKebabCase = (string) => {
   return string
-    .replace(/[^a-zA-Z0-9\. ]/g, '') //remove special chars
+    .replace(/[^a-zA-Z0-9\. ]/g, '') // remove special chars
```

Comment:
have u run linting? make sure to do so before committing.

Reply:
Yes, I did.
Unfortunately, it required a lot of fixes as the packages have not been updated for quite some time now.
I will address this in future PR.
Also will Add a workflow lint check too. WDYT?

Reply:
yes, lint workflow check upon/before PR should be good

Reply:
cool, we'll do that.